src/main/java/org/ecommerce/models/Response.java

```diff
@@ -24,11 +24,11 @@ public Response(boolean success, String message, HttpStatusCode statusCode) {
 
     @Override
     public String toString() {
-        return "ApiResponse{" +
-                "success=" + success +
-                ", message='" + message + '\'' +
-                ", data=" + data +
-                " statusCode=" + statusCode +
-                '}';
+        return "ApiResponse {" +
```

Comment:
Please try the lombok's `@ToString` annotation

Reply:
you're right, here's the commit https://github.com/algutierrez-griddynamics/e-commerce/pull/26/commits/ee36ea6e7896e19a17047c948f2395cfc7d5247f